content/en/blog/2022-10-14-unikraft-community-meetup.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "Unikraft Community Meetup '22"
+date = "2022-10-14T09:00:00+01:00"
+author = "Andrei Mutu"
+tags = ["community"]
++++
+
+### The First of Many
+
+We are excited to tell you about the first in-person Unikraft community meetup, a long awaited event which was painfully postponed by the pandemic.
+It took place in the shape of a three-day trip to Sinaia, Romania between the 14th and the 16th of October.
+
+This was a great opportunity for most of the core members to finally meet in-person, get to know one another more and exchange Unikraft war stories, all the while being introduced to Romanian customs.
+
+### A Hike, a Grill and a Presentation Walk into a Bar
+
+What do all these have in common, you ask?
+Well, one might be tempted to say nothing, but at a closer look this is what the main day of the trip looked like.
+
+After a late arrival in Sinaia our participants had a somewhat early and uneventful end of the day.
+Things took a more interesting turn when our participants went for a hike early in the morning (for those of you who, like me, can't believe it, we will have pictures at the end to prove it, so tag along).
+
+Some went for a more traditional hike at [Vf. Piscul Cainelui](https://muntii-nostri.ro/ro/routeinfo/sinaia-vf-piscul-cainelui), while others went for a more conservative visit of the local places, like the [Peles and Pelisor](https://peles.ro/) castle.
+
+Sometime at midday both groups returned to the accommodations, where the main event was about to start, a community contributors presentation, which was composed of five topics.
+
+The first one to break the ice was our community manager, [Răzvan Deaconescu](https://www.linkedin.com/in/razvandeaconescu/), who talked us through the general state of the community.
+The highlight of the presentation was the ever incresing number of github stars, recently breaking the thousand mark.
+For more details, we have a blogpost about it [here](https://unikraft.org/blog/2022-10-14-unikraft-reaches-1k-github-stars/).
+
+Coming immediately after, we had [Alexander Jung](https://www.linkedin.com/in/nderjung/), CPO at Unikraft, to give us a small talk about the current state of kraftkit, the future of the kraft companion for building unikernels, which we hope to deliver in the next release.
+Tune in for more details because we will for sure have a blogpost about it.
+
+Afterward, we had [Simon Kuenzer](https://www.linkedin.com/in/simon-kuenzer-690a3145/), CTO and Co-Founder, give us a presentation about musl support.
+Our aim through musl is to have a better binary compatibility, since musl is the closest to libc.
```

Comment:
```suggestion
Our aim through Musl is to have a better binary compatibility, since Musl is the closest to libc and provides a Linux / POSIX compatible interface.
```